### PR TITLE
New package: limine-3.5.5

### PR DESCRIPTION
**Pull request by mintsuki**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Comment by classabbyamp** [comment]
please don't close/reopen PRs, it just adds noise. you can do things like amend commits, rebase, and force-push instead.

**Comment by Chocimier** [comment]
>    I tested the changes in this PR: YES

How, give that package contains just license?

**Comment by mintsuki** [comment]
> How, give that package contains just license?

Fair enough, I was pretty sure I checked it worked, but I guess I was wrong. It should be okay now.

File: srcpkgs/limine/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'limine'
+pkgname=limine
+version=3.0.3
+revision=1
+archs="x86_64 x86_64-musl"
```
Comment (JamiKettunen) [inline srcpkgs/limine/template:5]: `x86_64*`, but it really only works on x86_64? What issues are there for e.g. `i686*`? It's marketed as a `x86/x86_64 BIOS/UEFI bootloader` after all in the `short_desc`

File: srcpkgs/limine/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#wrksrc=
```
Comment (JamiKettunen) [inline srcpkgs/limine/template:6]: Half of the variables in here are commented out, the commented ones should be removed just like in other templates.

**Review by JamiKettunen — CHANGES_REQUESTED** [review]

**Comment by mintsuki** [comment]
Both concerns addressed

File: srcpkgs/limine/template (hunk 1 above)
Comment (mobinmob) [reply srcpkgs/limine/template]: Works on i686, does not build on i686, Ι also have a WIP PR: https://github.com/void-linux/void-packages/pull/36380

Comment (mobinmob) [reply srcpkgs/limine/template]: > This builds and works on i686.

Oh, I see, you use the toolchain for cross-building. Nice :) Are you going to update the PR to the most recent version. I will close mine :)

Comment (mintsuki) [reply srcpkgs/limine/template]: I just updated the PR to version 3.4.4, and thanks for the interest in the project!

File: srcpkgs/limine/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+build_style=gnu-configure
+hostmakedepends="mtools nasm"
+short_desc="Advanced x86/x86_64 BIOS/UEFI bootloader"
+maintainer="Mintsuki <mintsuki@protonmail.com>"
+license="BSD-2-Clause"
+homepage="https://limine-bootloader.org/"
+distfiles="https://limine-bootloader.org/files/limine/limine-${version}.tar.gz"
+checksum=46edf68c36b98a277fb19564b46d295f2f2808dca0e1f01597f1d48709136bb6
+
+if [ "$XBPS_MACHINE" = "i686" ]; then
```
Comment (classabbyamp) [inline srcpkgs/limine/template:15]: what about the case where host=i686, target=i686?

Comment (mintsuki) [reply srcpkgs/limine/template:15]: Well that `if` should cover both cases, no?

**Comment by mintsuki** [comment]
Bump.

**Pull request closed without merging**